genrl/agents/bandits/contextual/common/base_model.py

```diff
@@ -2,11 +2,12 @@
 from typing import Dict
 
 import torch
-import torch.nn as nn
-import torch.nn.functional as F
+from torch.nn import functional as F
 
 from genrl.agents.bandits.contextual.common.transition import TransitionDB
 
+nn = torch.nn
+
```

Comment:
Is this a good way to write it? Seems sketchy. Saw that it was recurring.

Reply:
True, let me see if something can be done. LGTM throws alerts otherwise.

Reply:
The other way is to supress it the warning like `# noqa`. Which seems better to me than doing this. Switching to that.

Reply:
Although we do it the other way, seems like this should be the correct way. https://docs.python.org/2/reference/simple_stmts.html#import